refine

```diff
@@ -378,9 +378,9 @@ virtualenv_prepare() {
     
 # ----------------------------------------------------------------------------------------------
 
-mvn() {
-    mvn_prepare   
-    
+do_mvn() {
```

Comment:
I had to change the name of this function, as using `command -v` caused recursive infinite loop in the `test()` function because of a clash between the name of function and the `mvn` CLI command. (no idea why. I tried a bunch of other ways of fixing it, but renaming the function was the simplest and only one I could get working).